charts/gardener/templates/secret-default-domain.yaml

```diff
@@ -15,12 +15,17 @@ metadata:
   annotations:
     dns.garden.sapcloud.io/provider: {{ ( required ".controller.defaultDomains[].provider is required" $domain.provider ) }}
     dns.garden.sapcloud.io/domain: {{ ( required ".controller.defaultDomains[].domain is required" $domain.domain ) }}
+    {{- if ne $domain.provider "alicloud-dns" }}
     dns.garden.sapcloud.io/hostedZoneID: {{ ( required ".controller.defaultDomains[].hostedZoneID is required" $domain.hostedZoneID ) }}
+    {{- end }}
 type: Opaque
 data:
 {{- if eq $domain.provider "aws-route53" }}
   accessKeyID: {{ ( required ".controller.defaultDomains[].credentials.accessKeyID is required" $domain.credentials.accessKeyID ) | b64enc }}
   secretAccessKey: {{ ( required ".controller.defaultDomains[].credentials.secretAccessKey is required" $domain.credentials.secretAccessKey ) | b64enc }}
+{{- else if eq $domain.provider "alicloud-dns" }}
```

Comment:
Can't this be merged with line 23? Let's not duplicate this.

Reply:
The content in line 25 is not the same as line 28.